vpr/src/route/route_timing.cpp

```diff
@@ -1982,3 +1995,26 @@ static void init_net_delay_from_lookahead(const RouterLookahead& router_lookahea
         }
     }
 }
+
+//updates router iteration information and checks for breakpoints
+//stops if a breakpoint is encountered
+void update_router_info_and_check(char type, int net_id) {
+    t_draw_state* draw_state = get_draw_state_vars();
+    if (draw_state->list_of_breakpoints.size() != 0) {
+        if (type == 'r')
```

Comment:
enumerated type (BP_ROUTE_ITER and BP_NET_ID) would be more clear than 'r' and 'n' as flags.

Reply:
resolved